drivers/gpio/gpio_mchp_xec.c

```diff
@@ -9,13 +9,17 @@
 #include <errno.h>
 #include <device.h>
 #include <drivers/gpio.h>
+#include <sys/sys_io.h>
 #include <soc.h>
 
 #include "gpio_utils.h"
 
-#define XEC_GPIO_EDGE_DLY_COUNT		8
+#define XEC_GPIO_EDGE_DLY_COUNT		4
 /* read only register in same AHB segmment for dummy writes */
-#define XEC_GPIO_DLY_ADDR		0x40080150u
+#define XEC_GPIO_DLY_ADDR		\
+	((DT_REG_ADDR(DT_NODELABEL(pcr))) + MCHP_PCR_CLK_REQ0_OFS)
+
+#define XEC_ECIA_REGS ((struct ecia_regs *)(DT_REG_ADDR(DT_NODELABEL(ecia))))
```

Comment:
The define seems to be unused and not related to PR. Should be probably removed.

Reply:
@mnkp Changed done